src/qml/doc/icon-policy.md

```diff
@@ -0,0 +1,39 @@
+# Icon policy
+This document provides guidelines for contributing icons to Bitcoin Core.
+
+## Preparing Icons
+Both an icon source file, in Scalable Vector Graphics (SVG) format,
+and an optimized production file, in Portable Network Graphics (PNG) format,
+are required for each icon.
+
+#### SVG Source File
+SVGs are used as source files because they can scale while retaining image quality.
+They are not used in production due to limited application support.
```

Comment:
The phrase “limited application support.” sounds a little confusing and might be misleading. For example, one might think, which application are we talking about here? Is it “bitcoin core” or some other application that works on PNG, SVG, and other formats of images.

I think a better framing of this sentence would be:
“They are not used in production because they are not widely supported (compared to PNGs).”

In this framing, the reader will clearly understand that we are talking about format support in general.

Reply:
This line is just a quick explanation as to why we are not using SVG files as our production files. The reason for this is because of limited application support; this could refer to x, y, or z reason. 

In reality, we are/will be using SVG as production files because that is what designers have recommended. Getting into the particularities here, or what the reality of support for SVG files really is across all applications doesn't matter.

@shaavan: Changing from "limited application support" to "not widely supported" doesn't remove the inherent reference to 'outside' projects

I think the line is fine as is

Reply:
> Changing from "limited application support" to "not widely supported" doesn't remove the inherent reference to 'outside' projects

I can see what you are talking about. I guess the phrase "limited application support" is not as confusing as I first perceived it to be :)